packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js

```diff
@@ -20,6 +20,10 @@ class JWPlayer extends React.Component {
   }
 
   componentDidMount() {
+    const jwPlayerScript = document.getElementById('jw-player-script');
+    jwPlayerScript.onerror = function() {
+      this.props.onScriptError && this.props.onScriptError();
+    };
```

Comment:
don't know what to think about it, but refer to document seems not to be a clean solution :/

Reply:
that's where the script is attached 
https://github.com/micnews/react-jw-player/blob/master/src/react-jw-player.jsx#L53

BTW this is waiting for my PR on `react-jw-player` appending `onerror` properly
here: https://github.com/micnews/react-jw-player/blob/master/src/helpers/install-player-script.js#L5 without a fork

Reply:
We could prolly fork the `react-jw-player` repo and apply our changes while the PR in the original repo is not merged.
What do y'all think?

Reply:
with the latest modification I'm not sure the PR on `react-jw-player` is relevant.
And I rather add few lines on `molecule/video-player/jwplayer.js` than add a fork.